src/go/rpk/pkg/cli/cmd/cluster/config/lint.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	"gopkg.in/yaml.v3"
+)
+
+func newLintCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "lint",
+		Short: "Remove any deprecated content from redpanda.yaml.",
+		Long: `Remove any deprecated content from redpanda.yaml.
+
+Deprecated content includes properties which were set via redpanda.yaml
+in earlier versions of redpanda, but are now managed via Redpanda's
+central configuration store (and via 'rpk cluster config edit').
+
+This command is not for general changes to cluster configuration: use this only
+when redpanda will not start due to a configuration issue.
```

Comment:
Is it safe / fine to use this between releases if the cluster is down? Why would I not want to run this on a live cluster that is working perfectly after I upgrade?

Reply:
Oops, the second two paragraphs shouldn't have been there, they were incorrectly copied from the `reset` command.